docs/_layouts/default.html

```diff
@@ -53,10 +55,8 @@
         </div>
 
 
-        <!-- Concierge button anchored at the bottom of the LHN-->
-        <div class="floating-concierge-button">
-            <!-- Floating button content -->
-        </div>
+        <!-- Concierge button anchored at the bottom of the LHN (rendered on wider screens) -->
+        {% include floating-concierge-button.html id="floating-concierge-button-lhn" %}
```

Comment:
Ok, could you please explain why?

Reply:
IDs have to be unique to the page in order to be valid. So you run the risk of someone applying the ID somewhere else on the page to borrow styles or something. I like to reserve IDs for "functionality" (anchors, JS selectors) and let classes handle styles.

Reply:
I think in this context I'm going to disagree and say I prefer the use of ID. There are exactly two floating concierge buttons, and each has its own ID. The CSS selectors aren't used to control the basic style attributes they have in common (these are defined using the `floating-concierge-button` class).

The ID is used to show/hide the specific floating concierge buttons, which to me feels more like "functionality" than "styles", even though it's done via css instead of JavaScript.

Reply:
wfm